### ISSUE TITLE
java.lang.StackOverflowError

### ISSUE THREAD

**Issue by alexander-myltsev** [etorreborre/specs2#539]
I'm trying to update dependencies at https://github.com/GlobalNamesArchitecture/gnparser/compare/deps-updates

I then get the error while `sbt test`:

```
java.lang.StackOverflowError
	at org.specs2.control.eff.EffImplicits$$anon$2.ap(Eff.scala:183)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.Arrs.go$1(Eff.scala:379)
	at org.specs2.control.eff.Arrs.apply(Eff.scala:392)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2.ap(Eff.scala:185)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.Arrs.go$1(Eff.scala:379)
	at org.specs2.control.eff.Arrs.apply(Eff.scala:392)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2.ap(Eff.scala:185)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.Arrs.go$1(Eff.scala:379)
	at org.specs2.control.eff.Arrs.apply(Eff.scala:392)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2.ap(Eff.scala:185)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.Arrs.go$1(Eff.scala:379)
	at org.specs2.control.eff.Arrs.apply(Eff.scala:392)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9$$anonfun$apply$11.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2.ap(Eff.scala:185)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
	at org.specs2.control.eff.EffImplicits$$anon$2$$anonfun$ap$9.apply(Eff.scala:202)
```

How to fix it?

**Comment by etorreborre** [comment]
I don't know this from the top of my head. I'll have a look at that tonight.

**Comment by alexander-myltsev** [comment]
Also, if helps: https://circleci.com/gh/GlobalNamesArchitecture/gnparser/57?utm_campaign=build-timedout&utm_medium=email&utm_source=notification

**Comment by etorreborre** [comment]
This issue should now be fixed with [`3.8.7-20170202003034-3739406`](https://oss.sonatype.org/content/repositories/releases/org/specs2/specs2-common_2.11/3.8.7-20170202003034-3739406).
